Compiler/BackEnd/Initialization.mo

```diff
@@ -816,6 +816,9 @@ protected
   HashSet.HashSet hs;
   list<DAE.ComponentRef> crefs;
 algorithm
+  // lochel: workaround to align all elements
+  globalKnownVars := BackendVariable.listVar(BackendVariable.varList(globalKnownVars));
```

Comment:
Shouldn't there be a function BackendVariable.compressVars instead of varList+listVar?
My bet is that the new function will be more efficient in speed and memory...